atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -106,21 +108,32 @@ def __init__(self, tasker, workflow, kojihub, url,
         self.kojihub = kojihub
         self.koji_ssl_certs = koji_ssl_certs
         self.koji_proxy_user = koji_proxy_user
-
         self.koji_principal = koji_principal
         self.koji_keytab = koji_keytab
 
+        self.use_auth = use_auth
+        self.url = url
+        self.verify_ssl = verify_ssl
+
         self.blocksize = blocksize
         self.target = target
         self.poll_interval = poll_interval
 
         self.namespace = get_build_json().get('metadata', {}).get('namespace', None)
```

Comment:
This is now only used by `_get_openshift_session`, let's make `namesapce` a local variable there instead.

Reply:
ok